### PR TITLE
Conformance test suite for runtime support of checker environment

### PR DESCRIPTION
**Pull request by JimLarson**

### CODE REVIEW

File: tests/envcheck/testdata/go-0.1.0.textproto [hunk 1]
```diff
@@ -0,0 +1,1963 @@
+name: "go-0.1.0"
```
Comment (TristonianJones) [inline tests/envcheck/testdata/go-0.1.0.textproto]: DId you want to check the handling for go-0.2.0 instead, or is this in reference to the spec version?

File: tests/envcheck/envcheck.go [hunk 2]
```diff
@@ -0,0 +1,327 @@
+/*
```
Comment (TristonianJones) [inline tests/envcheck/envcheck.go]: Most of the time the doc strings are written as single line comments:

```
// first
// second
// ...
```
Possibly a bit annoying, but it is consistent with the majority of the code.

File: tests/envcheck/testdata/go-0.1.0.textproto (hunk 1 above)
Comment (JimLarson) [reply tests/envcheck/testdata/go-0.1.0.textproto]: It's a reference to the checker version, and we know that the go-0.2.0 checker has the same environment as go-0.1.0.

File: tests/envcheck/envcheck.go (hunk 2 above)
Comment (JimLarson) [reply tests/envcheck/envcheck.go]: Fixed.  Added copyright notice too.

**Pull request merged**